### PR TITLE
Hotline international number (EXPOSUREAPP-6011)

### PR DESCRIPTION
**Pull request by SamuraiKek**
Adds an extra international phone number for the "Technical Hotline" and "Request Tan" screens.
I've had to remove the "Call" button from the "Request TAN" screen because it wouldn't make sense to have it with two phone numbers in the UI. The behaviour is also more in line with the "Technical Hotline" screen.

- Technical Hotline number: +49 30 498 75401
- Request TAN number: +49 30 498 75402
- Make sure the right numbers are displayed on the right screens and when clicking the numbers, the dialer is prefilled with the corresponding pressed number.

### CODE REVIEW

**Comment by MikeMcC399** [comment]
Link back to issue [Hotline not reachable from abroad](https://github.com/corona-warn-app/cwa-documentation/issues/502) in https://github.com/corona-warn-app/cwa-documentation/issues/502.

**Review by MikeMcC399 — COMMENTED** [review]
Please review the layout, as the padding / margin above the text "Our customer service is here to help." has been effectively lost.

Hotline phone padding

**Comment by SamuraiKek** [comment]
> Please review the layout, as the padding / margin above the text "Our customer service is here to help." has been effectively lost.

Nice catch! I'll add some top margin.

**Comment by MikeMcC399** [comment]
@SamuraiKek 
That looks a lot better now!

BTW: Is it intended for there to be a similar-sounding branch with the same ID?
This PR is from the branch [feature/6011-hotline-international-number](https://github.com/corona-warn-app/cwa-app-android/tree/feature/6011-hotline-international-number).

I accidently checked out [feature/6011-international-hotline-number](https://github.com/corona-warn-app/cwa-app-android/tree/feature/6011-international-hotline-number) instead when I was trying to verify the updated fix. That confused me for a couple of minutes!

**Comment by SamuraiKek** [comment]
> @SamuraiKek
> That looks a lot better now!
> 
> BTW: Is it intended for there to be a similar-sounding branch with the same ID?
> This PR is from the branch [feature/6011-hotline-international-number](https://github.com/corona-warn-app/cwa-app-android/tree/feature/6011-hotline-international-number).
> 
> I accidently checked out [feature/6011-international-hotline-number](https://github.com/corona-warn-app/cwa-app-android/tree/feature/6011-international-hotline-number) instead when I was trying to verify the updated fix. That confused me for a couple of minutes!

Only the branch from this PR should be taken into consideration.
I first implemented it on version 2.0 on this branch  [feature/6011-international-hotline-number](https://github.com/corona-warn-app/cwa-app-android/tree/feature/6011-international-hotline-number) but this change should actually go in 2.1. So I've created a new branch based on 2.1 (the one in this PR). I did this to prevent rebasing my old branch on 2.1 and potentially messing something up. I'll delete the old branch based on 2.1 to avoid any confusion.

**Comment by MikeMcC399** [comment]
@SamuraiKek

> Only the branch from this PR should be taken into consideration.
> I first implemented it on version 2.0 on this branch [feature/6011-international-hotline-number](https://github.com/corona-warn-app/cwa-app-android/tree/feature/6011-international-hotline-number) but this change should actually go in 2.1. So I've created a new branch based on 2.1 (the one in this PR). I did this to prevent rebasing my old branch on 2.1 and potentially messing something up. I'll delete the old branch based on 2.1 to avoid any confusion.

I assume you mean "I'll delete the old branch based on **2.0** to avoid any confusion i.e. [feature/6011-international-hotline-number](https://github.com/corona-warn-app/cwa-app-android/tree/feature/6011-international-hotline-number), not the branch based on 2.1! 😄

**Comment by LukasLechnerDev** [comment]
image

This is the screen in technical hotline

Design:
- I think all the text in the screenshot should have the same font size/style. 

Text (according to figma)
- The text "Sprachen: " changed to "Der Kundenservice ist in folgenden Sprachen für Sie da:"
- The text "Der Anruf ist kostenfrei." in the bottom should be removed. 
- Some new newlines (\n)

-> these things also apply to the Request TAN screen

**Comment by SamuraiKek** [comment]
> This is the screen in technical hotline
> 
> Design:
> 
> * I think all the text in the screenshot should have the same font size/style.
> 
> Text (according to figma)
> 
> * The text "Sprachen: " changed to "Der Kundenservice ist in folgenden Sprachen für Sie da:"
> * The text "Der Anruf ist kostenfrei." in the bottom should be removed.
> * Some new newlines (\n)
> 
> -> these things also apply to the Request TAN screen

Latest commit should address this.

File: Corona-Warn-App/src/main/res/layout/include_navigation_row.xml [hunk 1]
```diff
@@ -13,52 +13,57 @@
...
 
     <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/navigation_row"
+        style="@style/phoneNumber"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content">
+        android:layout_height="wrap_content"
+        android:focusable="true">
+
+
```
Comment (LukasLechnerDev) [inline Corona-Warn-App/src/main/res/layout/include_navigation_row.xml:32]: Remove duplicate blank line 😉

**Review by LukasLechnerDev — APPROVED** [review]
LGTM! Thanks!

**Review by axelherbstreith — APPROVED** [review]

**Review by janetback — APPROVED** [review]
UA approved

**Pull request merged**